server/grpc_service.go

```diff
@@ -311,9 +313,10 @@ func (s *Server) RegionHeartbeat(stream pdpb.PD_RegionHeartbeatServer) error {
 
 		hbStreams := cluster.coordinator.hbStreams
 
-		if isNew {
+		if time.Since(lastBind) > heartbeatStreamRebindInterval {
+			regionHeartbeatCounter.WithLabelValues(storeLabel, "report", "bind").Inc()
 			hbStreams.bindStream(storeID, server)
```

Comment:
any side effect if you bindStream again after the interval?

Reply:
Just re-assign a map entry, should be trivial.